inc/thorvg.h

```diff
@@ -372,26 +372,9 @@ class TVG_API Paint
      *
      * @return Result::Success when the blending method is successfully set.
      *
-     * @BETA_API
```

Comment:
@hermet don't we want to add "@since..." ?

Reply:
@mgrudzinska updated, thanks